primitives/rpc/src/lib.rs

```diff
@@ -88,6 +89,18 @@ sp_api::decl_runtime_apis! {
 			nonce: Option<U256>,
 			estimate: bool,
 		) -> Result<fp_evm::CallInfo, sp_runtime::DispatchError>;
+		fn call(
+			from: H160,
+			to: H160,
+			data: Vec<u8>,
+			value: U256,
+			gas_limit: U256,
+			max_fee_per_gas: Option<U256>,
+			max_priority_fee_per_gas: Option<U256>,
+			nonce: Option<U256>,
+			estimate: bool,
+			access_list: Option<Vec<(H160, Vec<H256>)>>,
```

Comment:
Also this -- should be able to be changed to `AccessListItem` directly. The SCALE encoding is the same.

Reply:
Currently the pallet_evm runner expects the access list to be a tuple, that's why I left it un-updated.

Reply:
Also the evm expects the tuple, so the conversion needs to happen somewhere.